components/clubhouse_io/actions/create-story/create-story.js

```diff
@@ -0,0 +1,371 @@
+    completedAtOverride: {
+      type: "string",
+      label: "Completed at Override Date",
+      description:
+        "A manual override for the time/date the Story was completed.",
+      optional: true,
+    },
+    createdAt: {
+      type: "string",
+      label: "Created at Date",
+      description: "The time/date the Story was created.",
+      optional: true,
+    },
+    dueDate: {
+      type: "string",
+      label: "Due Date",
+      description: "The due date of the story.",
+      optional: true,
+    },
+    description: {
+      type: "string",
+      label: "Description",
+      description: "The description of the story.",
+      optional: true,
+    },
+    epicId: {
+      type: "integer",
+      label: "Epic ID",
+      description: "The unique identifier of the epic the story belongs to.",
+      async options() {
+        const options = [];
+        const epics = await this.clubhouse.listEpics();
+        epics.forEach((epic) => {
+          options.push({
+            label: epic.name,
+            value: epic.id,
+          });
+        });
+        return options;
+      },
+      optional: true,
+    },
+    estimate: {
+      type: "integer",
+      label: "Estimate",
+      description:
+        "The numeric point estimate of the story. Can also be null, which means unestimated.",
+      optional: true,
+    },
+    externalId: {
+      type: "integer",
+      label: "External Id",
+      description:
+        "This field can be set to another unique ID. In the case that the Story has been imported from another tool, the ID in the other tool can be indicated here.",
+      optional: true,
+    },
+    externalLinks: {
+      type: "string[]",
+      label: "External Links",
+      description: "A string array of External Links associated with this story.",
+      optional: true,
+    },
+    fileIds: {
+      type: "integer[]",
+      label: "File Ids",
+      description: "An integer array of IDs of files attached to the story.",
+      optional: true,
+    },
+    followerIds: {
+      type: "string[]",
+      label: "Follower Ids",
+      description: "A string array of UUIDs of the followers of this story.",
+      optional: true,
+    },
+    groupId: {
+      type: "string",
+      label: "Group Id",
+      description: "The id of the group to associate with this story. A group in Clubhouse API maps to a \"Team\" within the Clubhouse Product.",
+      optional: true,
+    },
+    iterationId: {
+      type: "integer",
+      label: "Iteration Id",
+      description: "The ID of the iteration the story belongs to.",
+      async options() {
+        const options = [];
+        const iterations = await this.clubhouse.listIterations();
+        iterations.forEach((iteration) => {
+          options.push({
+            label: iteration.name,
+            value: iteration.id,
+          });
+        });
+        return options;
+      },
+      optional: true,
+    },
+    labels: {
+      type: "string",
```

Comment:
No.
Labels passes to the API as an array of objects. 
Objects are supported as Proptypes but not array of objects as you can see in this image from Components API docs:
![image](https://user-images.githubusercontent.com/25184117/131558473-4eec1b5b-6c29-44a0-9d08-2fc8bbf247a0.png)

The only way an action can take arrays of objects as input is if the user references a previous object[] type value from a variable or previous step {{steps.some_step.obj_arr_value}} or sends an `string` that we can `JSON.parse` into an array of objects. This has been discussed with the team previously, for example in this [Slack discussion](https://pipedream-users.slack.com/archives/G01J50YGJGP/p1627605225000100?thread_ts=1627323038.043900&cid=G01J50YGJGP)

Reply:
@sergioeliot2039 good point about the objects, that makes sense. Until such time as Pipedream supports `object[]`, let's change this to support adding a single label. You can change the label to `Label` and the type to `object`. Be sure to document clearly in the description the properties that can be added to the label object. In the API call, you'll have to wrap the object in an array.

The reason for this is that it's not a good UX to ask Pipedream users to write JSON in their props, and I strongly suspect that it will be impractical in many use cases (consider trying to inject a value from a previous step that includes a character that would have to be escaped in a JSON string—currently, there's just no way to make that work, even with structured mode off.

Reply:
ok, for now I have made locally that changes,
what about the other props that ideally should be `object[]` types
i.e. comments
```
    comments: {
      type: "string",
      label: "Comments",
      description:
        "An array with comment objects to add to the story. Each comment object must have the [CreateStoryCommentParams](https://clubhouse.io/api/rest/v3/#CreateStoryCommentParams) structure. Alternatively, provide a string that will `JSON.parse` to an array of comment objects. ",
      default: "",
```
show we accept a simple object for all of these inestad of array?
other props with this situation are: `storyLinks`, `tasks1.

Reply:
@sergioeliot2039 for comments, it's a complex enough use case that I would not include any comment functionality in this component, and add a new component for creating a comment on a story.